README.md

```diff
@@ -1,2 +1,41 @@
 # Awesome Book
 
+## Built With
+
+- html
+- css
+- JavaScript
+- vscode
+- git
+- github
+
+## Getting Started
+
+- Add a .gitignore file.
+- Set up a linter for HTML,CSS and Javascript. Make sure that any files that are not relevant are ignored by git.
+- Add descriptive README file to your project - please use this template but remember to customize it to your project.
+- Actually, the webpage has two pages: Home and About pages.
+- The speaker section was created dynamically with javascript.
+- The mobile version has a hamburger menu for better navigation.
```

Comment:
- [OPTIONAL] As pointed out by the previous reviewer, this section should contain all the commands that are necessary to set up the project locally. Please let's consider fixing this so that our README file should be descriptive and professional. This will surely call the attention of someone viewing your project on Github.